packages/convertim/src/Model/Order/ConvertimOrderDataToOrderDataMapper.php

```diff
@@ -0,0 +1,190 @@
+        $this->mapConvertimTransportOrPaymentDataToOrderItem($convertimOrderData->getTransportData(), $orderData);
+
+        $this->cartFacade->deleteCart($cart);
+
+        return $orderData;
+    }
+
+    /**
+     * @param string $uuid
+     * @param \Shopsys\FrameworkBundle\Model\Order\Item\OrderItemData[] $orderItemDatas
+     * @return \Shopsys\FrameworkBundle\Model\Order\Item\OrderItemData
+     */
+    protected function getOrderItemByUuid(string $uuid, array $orderItemDatas): OrderItemData
+    {
+        foreach ($orderItemDatas as $orderItemData) {
+            if ($orderItemData->getItemUuid() === $uuid) {
+                return $orderItemData;
+            }
+        }
+
+        throw new OrderItemNotFoundException(sprintf('Order item with uuid "%s" not found.', $uuid));
+    }
+
+    /**
+     * @param string|null $countryName
+     * @return \Shopsys\FrameworkBundle\Model\Country\Country
+     */
+    protected function getCountryByName(?string $countryName): Country
+    {
+        $countryReturn = null;
+        $i = 1;
+
+        foreach ($this->countryFacade->getAllEnabledOnCurrentDomain() as $country) {
+            if ($i === 1) {
+                $countryReturn = $country;
+            }
+
+            if ($country->getName() === $countryName) {
+                return $country;
+            }
+
+            $i++;
+        }
+
+        return $countryReturn;
+    }
+
+    /**
+     * @param \Convertim\Order\ConvertimOrderTransportData|\Convertim\Order\ConvertimOrderPaymentData $convertimOrderTransportPaymentData
+     * @param \Shopsys\FrameworkBundle\Model\Order\OrderData $orderData
+     */
+    protected function mapConvertimTransportOrPaymentDataToOrderItem(
+        ConvertimOrderTransportData|ConvertimOrderPaymentData $convertimOrderTransportPaymentData,
+        OrderData $orderData,
+    ): void {
+        $transportOrderItemData = $this->getOrderItemByUuid($convertimOrderTransportPaymentData->getUuid(), $orderData->items);
+        $transportOrderItemData->totalPriceWithVat = Money::create($convertimOrderTransportPaymentData->getPriceWithVat());
+        $transportOrderItemData->totalPriceWithoutVat = Money::create($convertimOrderTransportPaymentData->getPriceWithoutVat());
+        $transportOrderItemData->unitPriceWithVat = Money::create($convertimOrderTransportPaymentData->getPriceWithVat());
+        $transportOrderItemData->unitPriceWithoutVat = Money::create($convertimOrderTransportPaymentData->getPriceWithoutVat());
+
+        if ($convertimOrderTransportPaymentData->getVatRate() > 0) {
+            $transportOrderItemData->vatPercent = Money::create(100)->divide((Money::create($convertimOrderTransportPaymentData->getPriceWithoutVat())->divide($convertimOrderTransportPaymentData->getVatRate(), 6))->getAmount(), 6)->round(6)->getAmount();
+        } else {
+            $transportOrderItemData->vatPercent = '0';
+        }
```

Comment:
Why do we need to calculate it separately? Isn't the result of  `$convertimOrderTransportPaymentData->getVatRate()` return the number we need? 

From docs:
![image](https://github.com/user-attachments/assets/1b2c983b-73c7-404e-b31d-11a36597f9b4)

Reply:
I have solved this by getting Vat from entities